db/schema.rb

```diff
@@ -341,6 +334,7 @@
     t.string "sexual_orientation"
     t.boolean "user_defined_symptom_onset"
     t.date "extended_isolation"
+    t.string "races", default: "--- []\n"
```

Comment:
Why YAML?

Reply:
serialized fields in Rails get stored as YAML strings. See `patient.rb` for where the field gets serialized.